mdpow/templates/NPT_opls.mdp

```diff
@@ -1,5 +1,5 @@
-; Gromacs mdp file
-; Template for MD in NPT with OPLS-AA (part of POW)
+; GROMACS mdp file
+; Template for MD in NPT with OPLS-AA (part of MDPOW)
```

Comment:
OPLS-AA .mdp files are also used for AMBER/GAFF, in which case the description will not be correct. I would put something like "OPLS-AA or AMBER".

Reply:
Maybe also change the name of the file into "NPT_opls_amber.mdp", which is more clear.

Reply:
I can change it. But wouldn't it be better to simply have separate files for the different force fields? I _hope_ that the non-bonded settings for OPLS-AA and AMBER/GAFF are indeed the same.

Reply:
Yes, it would be more clear to have separate files for AMBER/GAFF.

We take the parameters from the original force field files, so they should be correct (the difference in fudgeQQ is well known).

```
> cat /gem/iorga/repositories/2020-SAMPL7-PhysProp/03_logP_mdpow/sims/oplsaa/SM46/SM46/Equilibrium/water/top/pp_system.top | sed -n 25,27p
[ defaults ]
; nbfunc	comb-rule	gen-pairs	fudgeLJ	fudgeQQ
1		3		yes		0.5	0.5
> cat /gem/iorga/repositories/2020-SAMPL7-PhysProp/03_logP_mdpow/sims/gaff/SM46/SM46/Equilibrium/water/top/pp_system.top | sed -n 22,24p
[ defaults ]
; nbfunc        comb-rule       gen-pairs       fudgeLJ fudgeQQ
1               2               yes             0.5     0.8333

```

Reply:
made amber version